src/models/github.js

```diff
@@ -21,9 +21,9 @@ class GitHub extends Fetcher {
           title: item.title,
           body: item.body,
           state: item.state,
-          createdAt: new Date(item.created_at),
-          updatedAt: new Date(item.updated_at),
-          closedAt: item.closed_at ? new Date(item.closed_at) : null,
+          createdAt: item.created_at,
+          updatedAt: item.updated_at,
+          closedAt: item.closed_at ? item.closed_at : null,
```

Comment:
This line's a little funny now that we're not doing the date parsing here. Just `item.closed_at` would suffice.